julia.html.markdown

```diff
@@ -78,7 +78,7 @@ false
 1 < 2 < 3 # => true
 2 < 3 < 2 # => false
 
-# Strings are created with "
+# AbstractStrings are created with "
```

Comment:
Yeah, you're right, that is confusing as a first sighting for string in julia. I'll clarify it.